<?xml version="1.0" ?>
<launch>


<node name="TM_VS" pkg="franka_control_VS"  type="franka_control_TM_VS"  output="screen"/>

</launch>